WcaOnRails/app/models/competition.rb

```diff
@@ -567,7 +568,15 @@ def has_base_entry_fee?
   end
 
   def has_fees?
-    base_entry_fee_lowest_denomination + competition_events.sum(:fee_lowest_denomination) > 0
+    if base_entry_fee_lowest_denomination.nil?
+      competition_events.sum(:fee_lowest_denomination) > 0
+    else
+      base_entry_fee_lowest_denomination + competition_events.sum(:fee_lowest_denomination) > 0
+    end
+  end
+
+  def not_nil_fee?
```

Comment:
As discussed offline, I feel like `def entry_fee_required?` is a better name for this method.